modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Questions.es.js

```diff
@@ -551,9 +548,47 @@ export default withRouter(
 							section.actions.subscribe && (
 								<div className="c-ml-3">
 									<SectionSubscription
-										onSubscription={() =>
-											setSkipCache(true)
-										}
+										onSubscription={() => {
+											if (
+												sectionTitle &&
+												sectionTitle !== '0'
+											) {
+												deleteCacheKey({
+													operation: {
```

Comment:
any way of extracting this?

I think this is the best solution because we definitely do not want to skip cache in _getSections_ that gets called hundred of times (user subscriptions is fine). But conditionally skipping is also fine.

If there is a way of cleaning this and reusing the variables several times, I would go for it.

Reply:
Sure. I'll try to clean it and go for this solution